workflows/argo/chgres-cube.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: chgres-cube
+spec:
+  arguments:
+    parameters:
+    - name: restarts
+    - name: date
+    - name: source_resolution
+      value: C48
+    - name: target_resolution
+      value: C384
+    - name: tracers
+      value: '"sphum","liq_wat","o3mr","ice_wat","rainwat","snowwat","graupel","sgs_tke"'
+    - name: vcoord_file
```

Comment:
I wrote a simple Python tool that produces this file from a `fv_core.res.nc` file (so I can produce it for any vertical level configuration).  I could consider making that part of this workflow, but that would require adding a few Python dependencies.